x/marker/client/cli/tx.go

```diff
@@ -154,8 +154,6 @@ Valid Proposal Types (and associated parameters):
 			var proposal govtypesv1beta1.Content
 
 			switch args[0] {
-			case types.ProposalTypeAddMarker:
-				proposal = &types.AddMarkerProposal{}
 			case types.ProposalTypeIncreaseSupply:
```

Comment:
We should probably add a cli command to execute this proposal.  There is an example in the name module for  `ModifyNameProposal`.  This way we can set the authority as the gov module address easily.